library/common-library/README.md

```diff
@@ -32,8 +32,10 @@ Here is a list of global options:
 | global.licenseKey | licenseKey | `""` |  | This set this license key to use |
 | global.customSecretName | customSecretName | `""` |  | In case you don't want to have the license key in you values, this allows you to point to a user created secret to get the key from there |
 | global.customSecretLicenseKey | customSecretLicenseKey | `""` |  | In case you don't want to have the license key in you values, this allows you to point to which secret key is the license key located |
-| global.podLabels | podLabels | `{}` | yes | Additional labels for chart pods |
 | global.labels | labels | `{}` | yes | Additional labels for chart objects |
+| global.podLabels | podLabels | `{}` | yes | Additional labels for chart pods |
+| global.deploymentAnnotations | deploymentAnnotations | `{}` | yes | Annotations to add to deployments, statefulSets and daemonSets |
```

Comment:
here it states that we are also adding it to statefulSets, in the other docs those are not mentioned.

I would try to follow the same "UI" of labels